linen_examples/mnist/mnist.ipynb

```diff
@@ -0,0 +1,247 @@
+     },
+     "metadata": {}
+    }
+   ],
+   "source": [
+    "show_img_grid(\n",
+    "    [train_ds['image'][idx] for idx in range(25)],\n",
+    "    [f'label={train_ds[\"label\"][idx]}' for idx in range(25)],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Training"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stderr",
+     "text": "INFO:absl:Load dataset info from /Users/andstein/tensorflow_datasets/mnist/3.0.1\nINFO:absl:Field info.citation from disk and from code do not match. Keeping the one from code.\nINFO:absl:Reusing dataset mnist (/Users/andstein/tensorflow_datasets/mnist/3.0.1)\nINFO:absl:Constructing tf.data.Dataset for split train, from /Users/andstein/tensorflow_datasets/mnist/3.0.1\nINFO:absl:Constructing tf.data.Dataset for split test, from /Users/andstein/tensorflow_datasets/mnist/3.0.1\nINFO:absl:train epoch: 1, loss: 0.2708, accuracy: 91.90\nINFO:absl:eval epoch: 1, loss: 0.0516, accuracy: 98.30\n"
+    }
+   ],
+   "source": [
+    "# Train for a single epoch.\n",
+    "config.num_epochs = 1\n",
+    "optimizer = mnist_lib.train_and_evaluate(config, model_dir='./model')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Upload events to https://tensorboard.dev\n",
```

Comment:
We should probably stress here that data is PUBLIC when uploaded to tensorboard.dev